modules/react_paragraphs_behaviors/src/ReactBehaviorsPluginManager.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Drupal\react_paragraphs_behaviors;
+
+use Drupal\Component\Plugin\Exception\PluginException;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Extension\ThemeHandlerInterface;
+use Drupal\Core\Plugin\DefaultPluginManager;
+use Drupal\Core\Plugin\Discovery\ContainerDerivativeDiscoveryDecorator;
+use Drupal\Core\Plugin\Discovery\YamlDiscovery;
+
+/**
+ * Provides the default react_behaviors manager.
+ */
+class ReactBehaviorsPluginManager extends DefaultPluginManager implements ReactBehaviorsPluginManagerInterface {
+
+  /**
+   * Provides default values for all react_behaviors plugins.
+   *
+   * @var array
+   */
+  protected $defaults = [
+    'id' => '',
+    'label' => '',
+  ];
```

Comment:
Should we default the `bundles` option to include all paragraphs and rows?

Reply:
yeah. makes sense to me

Reply:
oh... wait i read that wrong... :thinking: I think it's better to make it explicitly required to prevent unintended auto enabling

Reply:
I'm fine either way.